generate.js

```diff
@@ -111,58 +108,45 @@ const populateRandomAmenities = num => {
 };
 
 const createRoom = () => {
-  const data = [];
-  for (let room = 1; room <= 100; room++) {
-    // average length for a sentence
-    const sentenceLength = faker.random.number({ min: 4, max: 10 });
 
+  out.write('room_id, city, type, title, max_guest, subtype, beds, baths, host_username, avatar, short_description, main_description, house_rules_description, highlights, house_rules, cancellations, sleeping_arrangements \n');
+
+  for (let room = 1; room < 2; room++) {
+
+    const sentenceLength = faker.random.number({ min: 4, max: 10 });
+    
+    let word = `${room},`;
```

Comment:
Maybe change this variable name from 'word' to 'sentence'? It's a little misleading because you're adding more words to the variable 'word' which is singular!